### ISSUE TITLE
native core klib file not found

### ISSUE THREAD

**Issue by zzmingo** [Kotlin/kotlinx.coroutines#784]
I got this error when running task compileKotlinIos.

```
exception: java.lang.IllegalStateException: Could not find "/Users/xxx/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlinx/kotlinx-coroutines-core-native_debug_ios_arm64/1.0.0/7ca0a37e5dca2e6edf4e605f87c8ea5e92d3ef77/kotlinx-coroutines-core-native.klib" 
in [/Users/xxx/Documents/labs/YuniMobile, /Users/xxx/.konan/klib, /Users/xxx/.konan/kotlin-native-macos-0.9.3/klib/common, /Users/xxx/.konan/kotlin-native-macos-0.9.3/klib/platform/ios_arm64].
```

But there is actually a file named kotlinx-coroutines-core-native.klib in that path.

My gradle config is:

```groovy
kotlin {
    targets {
        fromPreset(presets.android, 'androidLib') {
        }
        // This preset is for iPhone emulator
        // Switch here to presets.iosArm64 to build library for iPhone device
        fromPreset(presets.iosArm64, 'ios') {
            compilations.main {
                outputKinds 'FRAMEWORK'
//                cinterops {
//                    Alamofire {
//                        packageName 'framework.alamofire'
//                        extraOpts '-nodefaultlibs'
//                    }
//                }
            }
        }
    }
    sourceSets {
        commonMain {
            kotlin {
                srcDir "${rootDir.absolutePath}/src/generated"
            }
            dependencies {
                implementation project(":annotations")
                implementation 'org.jetbrains.kotlin:kotlin-stdlib-common'
                implementation 'org.jetbrains.kotlinx:kotlinx-coroutines-core-common:1.0.0'
            }
        }
        commonTest {
            dependencies {
                implementation 'org.jetbrains.kotlin:kotlin-test-common'
                implementation 'org.jetbrains.kotlin:kotlin-test-annotations-common'
            }
        }

        androidLibMain {
            dependencies {
                implementation 'org.jetbrains.kotlin:kotlin-stdlib'
                implementation 'org.jetbrains.kotlinx:kotlinx-coroutines-android:1.0.0'
            }
        }
        androidTest {
            dependencies {
                implementation 'org.jetbrains.kotlin:kotlin-test'
                implementation 'org.jetbrains.kotlin:kotlin-test-junit'
            }
        }
        iosMain {
            dependencies {
                implementation 'org.jetbrains.kotlin:kotlin-stdlib'
                implementation 'org.jetbrains.kotlinx:kotlinx-coroutines-core-native:1.0.0'
            }
        }
        iosTest {
        }
    }
}
```

Anyone help me?

------edit------
I was missing this warnning

```
warning: skipping /Users/zhangzhengming/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlinx/kotlinx-coroutines-core-native_debug_ios_arm64/1.0.0/7ca0a37e5dca2e6edf4e605f87c8ea5e92d3ef77/kotlinx-coroutines-core-native.klib. The compiler versions don't match. Expected '[0.9.3]', found '1.0.1-release-4583'
```

**Comment by CodeZilla7** [comment]
@zzmingo You haven't added the versions for the following dependencies:

- implementation 'org.jetbrains.kotlin:kotlin-stdlib-common'
- implementation 'org.jetbrains.kotlin:kotlin-test'
- implementation 'org.jetbrains.kotlin:kotlin-test-junit'
-  implementation 'org.jetbrains.kotlin:kotlin-test-common' 
- implementation 'org.jetbrains.kotlin:kotlin-test-annotations-common'

 Add the appropriate versions for each library you're using.

**Comment by zzmingo** [comment]
@CodeZilla7 Thanks, the version of stdlib seems will be appended automatically by applying plugin 'kotlin-multiplatform'. Anyway, let me have a try.

**Comment by zzmingo** [comment]
Close this, because it was fixed by 1.3.0-release
